src/style.css

```
@@ -0,0 +1,97 @@
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  font-weight: 400;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  font-weight: 900;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
```

for fixing horizontal scroll try to add such styles
```suggestion

  * {
    box-sizing: border-box;
  }
```